src/DependencyInjection/Configuration.php

```diff
@@ -17,10 +17,10 @@ public function getConfigTreeBuilder(): TreeBuilder
 
         $rootNode
             ->children()
-                ->scalarNode('url')->end()
+                ->scalarNode('url')->cannotBeEmpty()->end()
```

Comment:
This is like a BC break.. but `Meilisearch\Client` expects a string for the base url and if it was not set then `MeilisearchExtension` was setting argument to `null` which does not make sense.

Reply:
I agree. Not adding the URL means the library will not work. Unless somewhere else we have the `localhost` being passed automatically.

Reply:
Hm, we could set `http://localhost:7700` as default value then, at least it would not break app in compilation time

Reply:
It should be written somewhere then :)